dist/shiny-server-client.js

```diff
@@ -1953,9 +1954,14 @@ exports.createFactory = function (options) {
       }
     });
 
-    var conn = new global.SockJS(url, null, {
-      protocols_whitelist: whitelist
-    });
+    var transportDebugging = options.transportDebugging && options.transportDebugging == true;
```

Comment:
Yeah, that's a bit ugly, but it's an attempt to force a boolean value. I hate to blindly pass down a value that is _intended_ to be bool.